wiki/Beatmapping/Health_Drain/en.md

```diff
@@ -1,7 +1,99 @@
-# Health drain
+---
+tags:
+- HP
+- Passive HP
+- Drain
+---
 
-Health drain refers to the natural drain of the health bar that occurs when playing a [beatmap](/wiki/Beatmaps) while disregarding the influence of [hit objects](/wiki/Hit_Objects). This drain is dependent on the HP difficulty setting, which is set by the mapper of a beatmap. This value ranges from 0 to 10, where 10 is the highest and most punishing setting and 0 is the most forgiving. The HP difficulty setting can be changed alongside other difficulty settings in the [Beatmap Editor](/wiki/Beatmap_Editor). Doing this will change a Ranked beatmap's status to unranked and will erase all local scores on it.
+# Health
 
-osu!standard and osu!catch have health drain at a constant rate throughout the course of a beatmap, with the exception of breaks. As for osu!taiko and osu!mania, health drain is disabled, so only the hit objects will affect the health bar.
+Health or more commonly known as *HP*, is the value that decides if a player has failed or passed a [beatmap](/wiki/Beatmaps). Health can be seen in the health bar which updates as a beatmap progresses. Health is largely determined by the HP difficulty setting, which is set by the mapper of a beatmap.
 
-<!--TODO: Insert links to health bar, difficulty setting, mapper, ranked, unranked, modes -->
\ No newline at end of file
+The HP difficulty settting controls how heavily a player is penalized for missing notes and how much health is gained back by accurately hitting [hit objects](/wiki/Hit_Objects). HP values range from HP0 to HP10, where HP10 is the highest and most punishing setting and HP0 the most forgiving. The HP difficulty setting can be changed alongside other difficulty settings in the [Beatmap Editor](/wiki/Beatmap_Editor). Doing this will change a Ranked beatmap's status to unranked and will erase all local scores on it.<!--TODO: might be worth writing about this more in depth-->
+
+## Health drain
+
+A common system across game modes is known as health drain, which refers to the natural drain of the health bar that occurs when playing a beatmap. This value is also known as *HP drain*, and is the one mostly referred to when talking about health. Health drain is dependent on the HP difficulty setting, which is set by the mapper of a beatmap. HP values range from 0 to 10, where 10 is the highest and most punishing setting and 0 the most forgiving. On HP10 the health drains rapidly and a single miss may be enough to fail a player, whereas on HP0 multiple combos can be missed before failing a player.
```

Comment:
Probably don't need to discuss the difference between HP0 and HP10 in both Health and Health drain sections. I would remove

> Health drain is dependent on the HP difficulty setting, which is set by the mapper of a beatmap. HP values range from 0 to 10, where 10 is the highest and most punishing setting and 0 the most forgiving.

from here and keep the rest.